app/services/statistic_service.rb

```diff
@@ -4,7 +4,7 @@ def self.reporting(recruitment_cycle:)
       providers: ProviderReportingService.call(providers_scope: recruitment_cycle.providers),
       courses: CourseReportingService.call(courses_scope: recruitment_cycle.courses),
       publish: PublishReportingService.call(recruitment_cycle_scope: recruitment_cycle),
-      allocations: AllocationReportingService.call(recruitment_cycle_scope: recruitment_cycle),
+      allocations: AllocationReportingService.call(recruitment_cycle_scope: recruitment_cycle.previous),
```

Comment:
Use the `allocation_cycle_year` in the settings for all allocation related things

```suggestion
      allocations: AllocationReportingService.call(recruitment_cycle_scope: RecruitmentCycle.find_by(year: Settings.allocation_cycle_year)),
```

Reply:
Aweesome will do, thanks 🙌